docsrc/imap/support/feedback-meetings.rst

```diff
@@ -8,4 +8,4 @@ Join us online!
 
 Regular contributors catch up online weekly, both as a status checkpoint and to make sure we're all alive!
 
-Meetings are currently held at **Monday 11am UTC** via `Google Hangouts <https://plus.google.com/hangouts/_/g4xnqjjb5zvomzeb4kqvja3fz4a>`_. It's worth checking on :ref:`IRC <feedback-irc>` to confirm the time and URL if nobody else seems to be online.
+Meetings are currently held at **Monday 21:00 UTC** via `Google Hangouts <https://plus.google.com/hangouts/_/g4xnqjjb5zvomzeb4kqvja3fz4a>`_. It's worth checking on :ref:`IRC <feedback-irc>` to confirm the time and URL if nobody else seems to be online.
```

Comment:
This change is wrong, 21:00 is the time of the meeting in Melbourne, but it is 11:00 UTC.

Reply:
https://www.cyrusimap.org/dev/imap/support/feedback-meetings.html says 21 UTC.  The general problem, that needs to be approached is, that changes relevant for 3.0, 3.1 and perhaps 2.5 are usually applied only to one branch.

Reply:
Ahh, the master version is wrong.  I'll fix that up.

Yeah, it's a nuisance having the website split up the way it is!  It used to be its own repository, which was a different kind of nuisance.  This is an improvement, but not much of one :(